crates/worker/src/partition_processor_manager.rs

```diff
@@ -489,15 +489,15 @@ impl PartitionProcessorManager {
                             if let Err(err) = processor_state
                                 .on_leader_epoch_obtained(leader_epoch, leader_epoch_token)
                             {
-                                info!(%partition_id, "Partition processor failed to process new leader epoch: {err}. Stopping it now.");
+                                info!(%partition_id, %err, "Partition processor failed to process new leader epoch. Stopping it now.");
```

Comment:
I'm personally not a fan of having a period (`.`) at the end of every log message like this.

Reply:
Yes, will remove them.